### DISCUSSION TITLE
Stato delle fatture elettroniche inviate

### DISCUSSION THREAD

**Discussion by manuelzlp** [fattureincloud/api#327] [Q&A] [answered]
Buongiorno

Sto implementando la nuova API di FEiC però non riesco a trovare informazione riguardo lo stato delle fatture elettroniche inviate.

Ho provate ListIssuedDocuments / GetIssuedDocument e ritrovo tutti i dati della fattura (cliente, date, totali, ecc) però i campi 'e_invoice' e 'ei_status' mi tornano sempre null

C'è un'altra chiamata per queste informazioni?

Grazie!

**Comment by valmoz** [comment]
Buongiorno @manuelzlp 
deve usare il parametro fieldset:
https://developers.fattureincloud.it/docs/basics/additional-capabilities/customize-response

Grazie
Mauro

**Comment by manuelzlp** [comment, marked as answer]
Ciao!

Grazie, ho risolto aggiungendo 'detailed' come fieldset.

Per ultimo non riesco a trovare una lista dei possibili valori per 'ei_status', oltre a send, così da capire se fare la chiamata Get Rejection Reason. C'è una lista dei valori 'ei_status'?

Grazie ancora

**Reply by valmoz** [reply]
Buongiorno,
può trovarla nella nostra [API Reference](https://developers.fattureincloud.it/api-reference/#put-/c/-company_id-/issued_documents/-document_id-) cliccando su "Schema".

Gentilmente, le chiederei di aprire una nuova discussione se ha la necessità di fare nuove domande non collegate all'originale, in modo tale di permettere agli altri utenti di cercare più semplicemente le risposte fornite.

Le consiglio anche di provare a cercare tra le discussioni chiuse se per caso abbiamo già risposto alle sue domande.

Grazie
Mauro

**Reply by valmoz** [reply]
Buongiorno @manuelzlp abbiamo risposto ai suoi dubbi?

Grazie
Mauro

**Reply by manuelzlp** [reply]
Ciao Mauro, 

Tutto chiaro, grazie mille!

**Reply by valmoz** [reply]
Ottimo!

Buona giornata
